packages/kotti-ui/source/kotti-field-toggle/components/ToggleInnerSuffix.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+	<div class="kt-field__header">
+		<div class="kt-field__header__label">
+			<span :class="labelSuffixClasses" v-text="labelSuffix" />
+		</div>
+		<div
+			v-if="hasHelpText"
+			class="kt-field__header__help-text"
+			:class="iconClasses('header__help-text', ['interactive'])"
+		>
+			<FieldHelpText :helpText="helpText" :helpTextSlot="helpTextSlot" />
+		</div>
+	</div>
+</template>
+
+<script lang="ts">
+import { defineComponent, computed } from '@vue/composition-api'
+
+import FieldHelpText from '../../kotti-field/components/FieldHelpText.vue'
+import { useTranslationNamespace } from '../../kotti-i18n/hooks'
+import { KottiFieldToggle } from '../types'
+
+export default defineComponent({
+	name: 'ToggleInnerSuffix',
+	components: { FieldHelpText },
+	props: {
+		helpText: { default: null, type: String },
+		helpTextSlot: { default: () => [], type: Array },
+		hideValidation: { default: false, type: Boolean },
+		isEmpty: { default: false, type: Boolean },
+		isOptional: { default: false, type: Boolean },
+		validation: { default: () => ({}), type: Object },
+	},
+	setup(props: KottiFieldToggle.InnerSuffix) {
+		const validationType = computed(() => props.validation.type)
+		const showValidation = computed(
+			() => !(props.hideValidation || validationType.value === 'empty'),
```

Comment:
If `validation.value === 'empty'` we hide validation?

Reply:
This is correct, validation has `empty` value instead of `null`. See https://github.com/3YOURMIND/kotti/commit/692f636264e0f18f9049b3338c5b9127558d8c61